### ISSUE TITLE
TextField not removing characters on Samsung Device

### ISSUE THREAD

**Issue by Zazo032** [ueman/feedback#120]
**Describe the bug**
After re-opening the keyboard, the value seems to be "stuck", it can't be erased

**Version of the library**
2.0.0 (also 2.1.0).

**To Reproduce**
Steps to reproduce the behaviour:
1. Open feedback tool
2. Click on the text field
3. Write `aa` on the text field
4. Close keyboard
5. Click on the text field again
6. Try to remove the written text

**Expected behaviour**
The text is erased

**Actual behaviour**
The text is still shown

**Screenshots**
I'll try to record a video of the bug

**Logs **
```
[✓] Flutter (Channel stable, 2.5.0, on macOS 11.5.2 20G95 darwin-arm, locale
    en-DE)
    • Flutter version 2.5.0 at <elided>/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (35 hours ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at <elided>/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.56.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.23.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 93.0.4577.63

• No issues found!
```

**Additional context**
I was only able to reproduce the issue on a Samsung Galaxy A21s on Android 11 with Samsung Keyboard. The same code is working fine on a OnePlus 5 on Android 10 with Gboard.

**Comment by ueman** [comment]
Thanks for reporting this. I'll see if I can reproduce it.

**Comment by Zazo032** [comment]
https://user-images.githubusercontent.com/2807712/133247579-fa6d7297-0c48-49d9-a966-c5a3e7b8bed1.mp4

**Comment by ueman** [comment]
That's such a weird bug.

**Comment by Zazo032** [comment]
Managed to workaround it by providing a custom `feedbackBuilder` and placing `BetterFeedback` in the `MaterialApp`'s `builder` param.

**Comment by ueman** [comment]
I believe this is a Flutter or Samsung Keyboard Bug. If you use Gboard it works as expected.

**Comment by ueman** [comment]
Someone over at the linked issue on the Flutter repo posted a workaround/fix for this issue.

**Comment by ueman** [comment]
Fixed in 2.4.1

**Comment by pixotri** [comment]
Samsung Z fold 5 with Android 14. Samsung keyboard still has the issue, other keyboards don't have it.
Google pixel 6 with Android 14. With standard keyboard has the same issue.
Could you please fix it?
Checked on each version from 2.4.1 till 3.0.0
